lib/modules/onc_program_procedure.yml

```diff
@@ -1109,7 +1112,10 @@ procedure:
           not require a specific set of data to be loaded.  Because of this,
           omissions are not possible to detect.  However, the tester can use the
           tool to compare responses to what is displayed in the system, and
-          register this as an attestation.
+          register this as an attestation.  Additionally, US Core v3.1 does
+          not include two required USCDI v1 data elements for Patient Demographics
```

Comment:
Isn't it three elements?

Reply:
Yeah, there are two Patient elements and one Allergy element.  Good catch, thanks.  Fixed.